### DISCUSSION TITLE
'C:\Users\Name\AppData\Local\Temp\ScintillaNET\3.6.3\x64\SciLexer.dll'. ---> System.ComponentModel.Win32Exception (0x80004005): This program is blocked by group policy. For more information, contact your system administrator

### DISCUSSION THREAD

**Discussion by Moynzy** [MscrmTools/XrmToolBox#1167] [Q&A]
Good morning all, 

We have the XRMtoolbox on our estate and developers who have the "Portal Editor Tool" run in to the following: 
'C:\Users\test\AppData\Local\Temp\ScintillaNET\3.6.3\x64\SciLexer.dll'. ---> System.ComponentModel.Win32Exception (0x80004005): This program is blocked by group policy. For more information, contact your system administrator

I have Googled the error message, checked Event viewer under App Locker and cannot find what is blocking the program from accessing the file path. 

Can one anyone please advise on how to resolve? 

Kind Regards
Mo

**Comment by BetimBeja** [comment]
Hi @Moynzy ,
Did you contact your system administrator first?
` This program is blocked by group policy. For more information, contact your system administrator`

Just asking!

**Reply by Moynzy** [reply]
Hello BetimBeja, 

I am part of the Service Desk, so a 1/4 of a system admin, and have reached out to my escalation points, but we are getting nowhere. 

The only solution would I can think of is to provide Admin Rights, but this is a major security risk. 

Thanks
Mo
